packages/docs/static/definitions/processes/kycaml/0.0.1/generic--usa-indiv-accinv-all-checks

```diff
@@ -0,0 +1,20 @@
+{
+    "processName": "Accredited Investor",
+    "country": "USA",
+    "subjectType": "Individual",
+    "version": "0.0.1",
+    "basicChecks": [
+      "Legal Name",
+      "Date of Birth",
+      "Address",
+      "Government-issued Photo ID",
+      "Tax ID",
+      "Occupation",
+      "Source of Funds"
+    ],
+    "elegibilityChecks": [
+      "Assets",
```

Comment:
I think this should be something related to Net Worth, which technically would be `Assets - Liabilities`